src/_data/books-dart1.yml

```diff
@@ -63,34 +63,34 @@
     - Patricia Juberias
   publisher: "Moises Belchin & Patricia Juberias"
   cover: aprende-dart.jpg
-  link: "http://libroaprendedart.appspot.com/"
+  link: "https://libroaprendedart.appspot.com/"
   desc: "Aprende Dart is the first Spanish-language book about Dart. Aprende Dart es el primer libro en Español para programadores y diseñadores web que quieran conocer el lenguaje, aprender a manejar las herramientas y trabajar con las principales funciones y librerías. Conocerás la historia que hay tras el lenguaje y cómo surge, crearemos animaciones para interfaces gráficas web y trabajaremos con servicios web, veremos el potencial que nos brinda Dart en el lado servidor y comenzaremos a trabajar con Polymer.dart."
 - title: Dart for Absolute Beginners
   authors:
     - David Kopec
   publisher: Springer (Apress)
   cover: dart-for-absolute-beginners.jpg
-  link: "http://www.springer.com/us/book/9781430264811"
+  link: "https://www.springer.com/us/book/9781430264811"
   desc: "Dart for Absolute Beginners enables individuals with no background in programming to create their own web apps while learning the fundamentals of software development in a cutting edge language. Easily digested chapters, while comprehensive enough to explore the whole domain, are aimed at both hobbyists and professionals alike. The reader will not only gain an insight into Dart, but also the technologies behind the web. A firm foundation is laid for further programming studies."
 - title: Dart for Hipsters
   authors:
     - Chris Strom
   publisher: Pragmatic Publishers
   cover: dart-for-hipsters.png
-  link: "http://pragprog.com/book/csdart/dart-for-hipsters"
```

Comment:
This link didn't lead to the book and I couldn't find a functional link on their website, so I updated the link to Google Books which has further information.